django/contrib/admin/options.py

```diff
@@ -1946,6 +1946,20 @@ def history_view(self, request, object_id, extra_context=None):
             "admin/object_history.html"
         ], context)
 
+    def get_formset_params(self, request, obj, inline, prefix):
```

Comment:
I think we'd call this `get_formset_kwargs()` normally.

Reply:
Sure!
I will change the name of this function with the commit for documentation.